### PR TITLE
switch to using public attributes

### PR DESCRIPTION
**Pull request by DeNeutoy**
Not much going on here really.

### CODE REVIEW

**Comment by joelgrus** [comment]
what's the motivation for this? do we expect that people using the library will need access to those?

File: allennlp/data/fields/text_field.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def get_padding_lengths(self) -> Dict[str, int]:
 
     # @overrides
```
Comment (matt-gardner) [inline allennlp/data/fields/text_field.py]: Missed this one.

File: allennlp/data/fields/text_field.py [hunk 2]
```diff
@@ -77,7 +77,7 @@ def get_padding_lengths(self) -> Dict[str, int]:
...
     def sequence_length(self) -> int:
-        return len(self._tokens)
+        return len(self.tokens)
 
     # @overrides
```
Comment (matt-gardner) [inline allennlp/data/fields/text_field.py]: And this one.

**Comment by matt-gardner** [comment]
Motivation: this fixes #145.  Given that we want public access to these fields (we made accessors for them), seems more pythonic to just have the fields be public.

**Comment by matt-gardner** [comment]
@joelgrus, do you agree?

**Comment by joelgrus** [comment]
yes, looks good

**Pull request merged**